src/lib/core/fiber.c

```diff
@@ -2357,13 +2357,22 @@ fiber_set_system(struct fiber *f, bool yesno)
 	}
 }
 
+void
+fiber_set_managed_shutdown(struct fiber *f)
+{
+	assert(f->flags & FIBER_IS_SYSTEM);
```

Comment:
Let's drop the assertion - nothing bad happens if someone sets this flag on a non-system fiber, really.

Reply:
Fixed.